### ISSUE TITLE
Auto play issues

### ISSUE THREAD

**Issue by VGisHere** [AdguardTeam/BlockYouTubeAdsShortcut#10]
Can auto play be made optional?
The issue I noticed is as follows -
Play video with AdGuard extension
Open new tab
Play another video with AdGuard extension
1st video stops, expected but sometimes may not be desirable
On closing the new video tab, if tab switched to old video, old video starts playing but the play button still shows status as paused. And it takes a couple of taps if old video has to be paused.

TLDR: request to make autoplay optional or be removed.
